### PR TITLE
(chore) Prepare for 2.7.0 release

### PR DESCRIPTION
**Pull request by yasirfolio3**
## Summary
- Update changelog for a new minor release 2.7.0 - UserProfileService

## Issues
- https://optimizely.atlassian.net/browse/OASIS-8254

### CODE REVIEW

**Review by jaeopt — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.7.0] - March 28, 2022
+
+- Added `UserProfileService` support. Out of the box implementations include `in-memory`, `rest` and `redis`. In-memory service supports both `fifo` and `lifo` orders. ([#326](https://github.com/optimizely/agent/pull/326)).
```
Comment (Mat001) [inline CHANGELOG.md:9]: Change "Added" to "Add" (also below).
And change "Updated" to "Update" (below).

We've been using directive verbs, not past tense. To be consistent how verbs have been used.

**Review by Mat001 — CHANGES_REQUESTED** [review]

**Review by Mat001 — APPROVED** [review]
LGTM

**Review by jaeopt — APPROVED** [review]
LGTM with a nit

**Pull request merged**